download-cesky-rozhlas-junior.sh

```diff
@@ -26,9 +26,9 @@ fi
 # And now process it all
 while IFS= read -r line
 do
-  url="$(echo ${line}| jq -r '.href')"
+  url="$(echo """${line}"""| jq -r '.href')"
   # Neuter the name a bit, even though it could be better
-  name="$(echo ${line} | jq -r '.name' | tr ': .' '___')"
+  name="$(echo """${line}""" | jq -r '.name' | tr ': .' '___')"
```

Comment:
Proc jsou tu ty """ ?? 3x " ???

Reply:
Pokud jsem se zásadně nespletl, je to způsob, jak tam dostat uvozovky, což chtěl shellcheck. Jestli to dělá přesně to, co bych chtěl, to nevím, ale přinejmenším to funguje.